addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -6053,6 +6053,40 @@ QUnit.module("Views", (hooks) => {
         assert.strictEqual(form.env.services.router.current.hash.id, 2);
     });
 
+    QUnit.test("switching to non-existing record", async function (assert) {
+        assert.expectErrors();
+
+        patchWithCleanup(browser, {
+            setTimeout(fn) {
+                return fn(); // update the router hash directly
+            },
+        });
+        const form = await makeView({
+            type: "form",
+            resModel: "partner",
+            serverData,
+            arch: '<form><field name="foo"></field></form>',
+            resId: 1,
+            resIds: [1, 999, 2],
+        });
+
+        assert.strictEqual(target.querySelector(".o_pager_value").textContent, "1");
+        assert.strictEqual(target.querySelector(".o_pager_limit").textContent, "3");
+        assert.strictEqual(form.env.services.router.current.hash.id, 1);
+
+        await click(target.querySelector(".o_pager_next"));
+        assert.verifyErrors(["Can't fetch record(s) 999 on model partner. They might have been deleted."]);
```

Comment:
suggestion (as we're waiting for @xmo-odoo to override the ci security anyway): have the fetch error handler alive and check that the notication is displayed (i.e. no error, just a notification)

Reply:
The error is catch by the expectErrors, so there is no notification. What would be the best ? Not using expectErrors and do the same thing that in the test load_state_tests.js or keeping the expectError and just test the notification in the other test (loas_state_tests.js) ?